data_node/entrypoint/main.go

```diff
@@ -1,10 +1,22 @@
 package main
 
 import (
+	"log"
+	"net/http"
+
 	datanode "github.com/SayedAlesawy/Videra-Storage/data_node"
+
+	"github.com/julienschmidt/httprouter"
 )
 
 func main() {
 	dataNode := datanode.NodeInstance()
 	dataNode.JoinCluster()
+
+	um := datanode.NewUploadManager()
+
+	router := httprouter.New()
+	router.POST("/upload", um.HandleUpload)
+
+	log.Fatal(http.ListenAndServe(":8080", router))
```

Comment:
We can we move the port to an env var? Check `conifg/data_node.go`. Just add a new `PORT` env var and add `8080` as it's fallback value.